compiler/rustc_hir_typeck/src/fallback.rs

```diff
@@ -506,14 +506,18 @@ impl<'tcx> FnCtxt<'_, 'tcx> {
         // then this code will be broken by the never type fallback change.qba
         let unit_errors = remaining_errors_if_fallback_to(self.tcx.types.unit);
         if unit_errors.is_empty()
-            && let never_errors = remaining_errors_if_fallback_to(self.tcx.types.never)
-            && !never_errors.is_empty()
+            && let mut never_errors = remaining_errors_if_fallback_to(self.tcx.types.never)
+            && let [ref mut never_error, ..] = never_errors.as_mut_slice()
```

Comment:
Do you think we should point out?

Or maybe I should make it a subdiagnostic so we can have multiple of them?

If we print out >1, I should probably do some deduplication like we do for real fulfillment errors 🤔

Reply:
I think ideally we _should_ print out all the errors, but maybe it will be too noisy, idk. This PR as is seems like a good first step/clear improvement.